mobile/mobile_45.asm

```diff
@@ -5026,27 +5024,27 @@ endr
 	jp nz, .asm_115c64
 	ret
 
-Function115cfd:
-	cp $2b
+Function115cfd: ; Base64
```

Comment:
Is this encoding or decoding Base64? Please change the `Function115cfd` label to clarify.